cogs/mod.py

```diff
@@ -45,6 +47,37 @@ def __init__(self, bot):
             await ctx.send(default.actionmessage("kicked"))
         except Exception as e:
             await ctx.send(e)
+            
+    @commands.has_permissions(administrator=True)
+    @commands.command()
+    async def say(self, ctx, *, msg=None):
+        if ctx.message.attachments:
+            file = BytesIO()
+            att = ctx.message.attachments[0]
+            await att.save(file)
+            file.seek(0)
+            await ctx.send(msg, file=discord.File(file, filename=att.filename))
+        else:
+            await ctx.send(msg)
+        await ctx.message.delete()
+    
+    @commands.command()
+    @commands.has_permissions(ban_members=True)
+    async def softban(self, ctx, member: str, *, reason: str=None): 
+        await member.ban(reason=reason)
+        await member.unban(reason=reason)
+        await ctx.send(f'Done. {member.name} was softbanned.') 
+        
+    
+    @commands.command()

```

Comment:
There's already a command for such [here](https://github.com/AlexFlipnote/discord_bot.py/blob/master/cogs/mod.py#L216)